src/main/java/com/scaleset/cfbuilder/ec2/metadata/CFNCommand.java

```diff
@@ -0,0 +1,46 @@
+package com.scaleset.cfbuilder.ec2.metadata;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@JsonPropertyOrder({"command", "env", "cwd"})
+public class CFNCommand {
+
+    @JsonProperty("command")
+    public String command;
```

Comment:
Do these need to be public? Please also check the access modifiers for the other CFN classes.

Reply:
Yes these need to be public because otherwise the jackson module cant't access the attributes. Though one could implement this with private attributes and getters with annotations